### ISSUE TITLE
The apparently ubiquitous "No Frame Was Found In The Output From Blender" issue.

### ISSUE THREAD

**Issue by shred787** [OpenShot/openshot-qt#3899]
I have literally tried everything I can find online to try to get this resolved.

I have upgraded everything. I ran openshot as Administrator. I have checked my path to the Blender executable. The works.

Running on Win7. 

Does anyone actually have a fix for this? It seems this is pretty foundational to the use of Openshot and the value statement of the program is substantially decreased if this isn't resolved.

**Comment by ferdnyc** [comment]
@shred787 

Please provide the following details, at a minimum:
* What version of OpenShot you're running
* What version of Blender you're using

Uploading a copy of your <code>C:\Users\\<var>username</var>\\.openshot_qt\openshot-qt.log</code> would also help, as there should be logs of the Blender session attempt.

If you're running 2.5.1-dev2 or later, you can also open a Command Prompt window and run OpenShot from there with `--debug`, that way you'll see every line of Blender output. (Please post the results, if you do.):
```command
cd "C:\Program Files\OpenShot Video Editor\"
openshot-qt-cli.exe --debug
```
(I _think_ that's the right directory to be in, if not just look around for that executable.)

**Comment by Cuppit** [comment]
I was running into the same error, and thanks to the logs (C:\Users\username\.openshot_qt\openshot-qt.log) I was able to solve it! (in my case, at least)  At the end of the logs was the following line:

`imb_savepng: Cannot open file for writing: 'C:\Users\<path to local user blender assets>\My "Animated" Title.png'`

The File Name turned out to be my problem this time.  I had quotation marks and spaces in the File Name title.   So I changed 'My "Animated" Title' to 'myanimatedtitle' in the 'File Name' field in OpenShot's "Animated Titles (Powered by Blender)" window when I went to make the 3D title card, and it rendered!

Hope this saves someone else a headache down the line.
